Gruntfile.coffee

```diff
@@ -1240,6 +1240,16 @@ module.exports = (grunt) ->
 					"theme/**/*.js"
 					"tasks/*.js"
 				]
+			travis:
+				options:
+					configFile: '.travis.eslintrc.json'
+					quiet: true
+				all:
+					src: [
+						"src/**/*.js"
+						"theme/**/*.js"
+						"tasks/*.js"
+					]
```

Comment:
Actually the block above is the first step. The second step is removing the .travis.eslintrc.json and changing the block with:

```coffee
travis:
	options:
		rules: do ->
  			rules = grunt.file.readJSON('.eslintrc.json')
  			rules['linebreak-style'] = 2
  			rules
		quiet: true
	src: "<%= eslint.all.src %>"
```